rosidl_generator_type_hash/resource/IndividualTypeDescription.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "IndividualTypeDescription.schema.json",
+  "title": "IndividualTypeDescription",
+  "description": "type_description_interfaces/msg/IndividualTypeDescription.msg JSON representation",
+  "type": "object",
+  "properties": {
+    "type_name": {"type": "string", "maxLength": 255},
+    "fields": {
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/Field"
+      }
+    }
+  },
+  "required": ["type_name", "fields"],
+  "additionalProperties": false,
+  "$defs": {
+    "Field": {
+      "type": "object",
+      "properties": {
+        "name": {"type": "string"},
+        "type": {"$ref": "#/$defs/FieldType"}
+      },
+      "required": ["name", "type"],
+      "additionalProperties": false
+    },
+    "FieldType": {
+      "type": "object",
+      "properties": {
+        "type_id": {"type": "integer", "minimum": 0, "maximum": 255},
+        "length": {"type": "integer", "minimum": 0},
```

Comment:
are you able to add `description` fields to these? i'm confused between `length` and `string_length`.

Reply:
I'm hesitant to copy any more information than is strictly necessary. It says up top `"description": "type_description_interfaces/msg/IndividualTypeDescription.msg JSON representation"` so in my mind all metadata like descriptions should be referred to from there (https://github.com/ros2/rcl_interfaces/blob/type_description_interfaces/type_description_interfaces/msg/FieldType.msg)

Reply:
:+1: missed that.